conans/test/functional/toolchains/test_nmake_toolchain.py

```diff
@@ -13,7 +13,8 @@
     [
         ("msvc", "190", "dynamic", "14", "Release", [], [], [], [], []),
         ("msvc", "190", "dynamic", "14", "Release",
-         ["TEST_DEFINITION1", "TEST_DEFINITION2=0", "TEST_DEFINITION3=", "TEST_DEFINITION4=TestPpdValue4"],
+         ["TEST_DEFINITION1", "TEST_DEFINITION2=0", "TEST_DEFINITION3=", "TEST_DEFINITION4=TestPpdValue4",
```

Comment:
Missing assert/checks in test

Reply:
TIL https://github.com/conan-io/conan/blob/694b7d1c7e72c4cf39ac9bc8c6f5b0697f8aa82a/conans/test/functional/utils.py#L140 this is already checked! No need to manually do that